general/okina.hpp

```diff
@@ -36,15 +40,37 @@
 template <size_t BLOCKS, typename DBODY, typename HBODY>
 void wrap(const size_t N, DBODY &&d_body, HBODY &&h_body)
 {
-   const bool gpu = mfem::config::usingGpu();
-   if (gpu)
+   const bool gpu  = mfem::config::usingGpu();
+   const bool raja = mfem::config::usingRaja();
+
+   if (gpu && raja)
+   {
+#ifdef MFEM_USE_RAJA
+      return RAJA::forall<RAJA::cuda_exec<BLOCKS>>(RAJA::RangeSegment(0,N), d_body);
+#else
+      MFEM_ABORT("RAJA requested for MFEM but RAJA is not enabled!");
+#endif
+
+   }
+   else if (gpu)
    {
       return cuWrap<BLOCKS>(N,d_body);
    }
+   else if (!gpu && raja)
+   {
+
+#ifdef MFEM_USE_RAJA
+      return RAJA::forall<RAJA::loop_exec>(RAJA::RangeSegment(0,N), h_body);
+#else
+      MFEM_ABORT("RAJA requested for MFEM but RAJA is not enabled!");
+#endif
+
+   }
    else
    {
       for (size_t k=0; k<N; k+=1) { h_body(k); }
    }
+
```

Comment:
@jdahm @camierjs , RAJA also supports OpenMP and TBB backends. We could always add additional if statements, but wasn't sure if there was a better way. What do you guys think?